src/client/components/simple-table/simple-table.tsx

```diff
@@ -115,12 +117,13 @@ export class SimpleTable extends React.Component<SimpleTableProps, SimpleTableSt
   }
 
   renderRow(row: any, columns: SimpleTableColumn[], index: number): JSX.Element {
+    const { hoveredRowIndex } = this.state;
     var items: JSX.Element[] = [];
 
     for (let i = 0; i < columns.length; i++) {
       let col = columns[i];
 
-      let icon = col.cellIcon ? <SvgIcon className="icon" svg={require(`../../icons/${col.cellIcon}.svg`)}/> : null;
+      let icon = col.cellIcon ? <SvgIcon svg={require(`../../icons/${col.cellIcon}.svg`)}/> : null;
```

Comment:
So I know this is not new code but there is a slight improvement that can be made here: webpack is smart(ish) when it sees code like `require(`../../icons/${col.cellIcon}.svg` it thinks to itself ok cool this might be any file that is `../../icons/*.svg` better depend on all of them to be safe.
It would be better to have a prefix for the icons. Something like: `require(`../../icons/table-${col.cellIcon}.svg` (and rename the icon files accordingly) that way only files with this prefix will be dependent on here. see: https://github.com/implydata/pivot/blob/master/src/client/components/vis-selector-menu/vis-selector-menu.tsx#L88